spark/src/main/java/org/opensearch/sql/spark/dispatcher/SparkQueryDispatcher.java

```diff
@@ -200,4 +243,44 @@ private static Map<String, String> getDefaultTagsForJobSubmission(
     tags.put(DATASOURCE_TAG_KEY, dispatchQueryRequest.getDatasource());
     return tags;
   }
+
+  @Getter
+  @RequiredArgsConstructor
+  public static class DropIndexResult {
+    private static final int PREFIX_LEN = 4;
+    private static final String DELIMITER = ";";
+
+    private final String status;
+    private final String errorMsg;
+
+    public static DropIndexResult fromJobId(String jobId) {
+      String[] results =
+          new String(Base64.getDecoder().decode(jobId)).substring(PREFIX_LEN).split(DELIMITER);
+      return new DropIndexResult(results[0], results[1]);
+    }
+
+    public String toJobId() {
+      String queryId =
+          RandomStringUtils.randomAlphanumeric(PREFIX_LEN)
+              + String.join(DELIMITER, status, errorMsg);
+      return Base64.getEncoder().encodeToString(queryId.getBytes(StandardCharsets.UTF_8));
```

Comment:
What is the length of this result? 
If it is more or less similar to emrJobId it would be good.

Reply:
`SXlRbFhVczI1bEZBSUxFRA==` similar